src/data/posts/twitter-and-the-perils-of-obedience.mdx

```diff
@@ -0,0 +1,164 @@
+
+This is a very real, very observable truth: countless racists, abusers, and other dangerous people are being given "amnesty" to come back the platform.
+
+I've seen many others abandon their twitter accounts in the wake of these changes, but -- friends -- I'm honestly shocked that so many people choose to remain.
+
+Why are so many people still on twitter, administer electric shocks to the collective public by keeping their accounts alive?
+
+I'm reminded by the words of Stanley Milgram, in his reflection on the results of his now-infamous experiment:
+
+> Ordinary people, simply doing their jobs, and without any particular hostility on their part, can become agents in a terrible destructive process. Moreover, even when the destructive effects of their work become patently clear, and they are asked to carry out actions incompatible with fundamental standards of morality, relatively **few people have the resources needed to resist authority.**
+
+For many people, Twitter was a place to connect with friends, family, and colleagues. I know many people with follower counts in the tens of thousands, who have built a career around their twitter presence. I can understand a deep hesitancy to abandon a platform that has been so central to your life.
+
+But fam, seriously -- **if not now, when?**
+
+## A call to action
+
+Friends, it's time to leave Twitter. Shutter your accounts, delete the app, and move on. I know it's hard, but it's time. Every moment you spent on Twitter is feeding legitimacy to a platform that is enabling abusers, endangering global politics, and destroying the fabric of our society.
+
+It is no longer acceptable to stay aboard the ship as it sinks - to stay active on Twitter despite their new policies and leadership is a vanity, a bath in your own privilege.
+
+Bring your voice and your conversations elsewhere, and send the message that with its current leadership, Twitter is no longer a place for you.
+
+<Image
+  publicId="posts/twitter-and-the-perils-of-obedience/experiment-1"
+  alt=""
+/>
+<Image
+  publicId="posts/twitter-and-the-perils-of-obedience/experiment-2"
+  alt=""
+/>
+<Image
+  publicId="posts/twitter-and-the-perils-of-obedience/experiment-3"
+  alt=""
+/>
+
+<br />
+<Colophon />
+
+## Footnote: on the ethics of the Milgram Experiment
+
+It's important to note that in the years since Milgram Experiment was conducted, it has been criticized for its methodology. [Research Ethics](https://en.wikipedia.org/wiki/Research#Research_ethics) is a complex and nuanced topic, and I don't believe that the experiment would be approved by modern ethics boards in its original form. However, I believe that the results of the experiment are still relevant, and that the experiment itself is still a valuable tool for understanding human behavior.
```

Comment:
The later study made some modifications to get by ethics boards. Like lowering the level of the simulated shock and sceening candidates to find those who might be adversely affected.